LandingPage.css

```diff
@@ -0,0 +1,97 @@
+body {
+  font-family: 'Roboto', sans-serif;
+  margin: 0;
+  padding: 0;
+  background-color: #f5f3f5;
+  color: #38023b;
+  line-height: 1.6;
+}
+
+.navBar {

```

Comment:
in general css prefers hyphens rather than camelcase so `nav-bar` instead of `navBar`